content/en/docs/installing-on-kubernetes/hosted-kubernetes/install-ks-on-aks.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Deploy KubeSphere on AKS"
+keywords: "kubesphere, kubernetes, docker, Azure, AKS"
+description: "How to deploy KubeSphere on AKS"
+---
+
+This guide walks you through the steps of deploying KubeSphere on Azure Kubernetes Service.
```

Comment:
You'd better add a website link for `Azure Kubernetes Service`

Reply:
ok